lib/dues_calculator.rb

```diff
@@ -1,12 +1,6 @@
 # frozen_string_literal: true
 
 module DuesCalculator
-  def self.update_exchange_rates_if_needed
-    if !Money.default_bank.rates_updated_at || Money.default_bank.rates_updated_at < 1.day.ago
-      Money.default_bank.update_rates
-    end
-  end
```

Comment:
Why does this need to go? Especially for `CurrencyLayer` which is rate-limited (we pay to get a maximum amount of 10,000 requests per month) having a caching mechanism like that would be lovely.

Reply:
Including the details in the other comment.